src/cascadia/TerminalApp/Pane.h

```diff
@@ -200,6 +219,19 @@ class Pane : public std::enable_shared_from_this<Pane>
 
     static void _SetupResources();
 
+    struct PanePoint
+    {
+        float x;
+        float y;
+    };
+
+    struct FocusNeighborSearch
+    {
+        std::shared_ptr<Pane> focus;

```

Comment:
okay, this is gonna seem like a nit: Could we rename this member to `current`? I think that might help the readability, in places like

```c++
        if (_IsAdjacent(focus.focus, focus.focusOffset, shared_from_this(), offset, direction))
```

That would become

```
        if (_IsAdjacent(focus.current, focus.focusOffset, shared_from_this(), offset, direction))
```

Reply:
Maybe this particular example would be improved be renaming the local variable to something like `searchResult`? That way we have code that looks like
```        if (_IsAdjacent(searchResult.focus, searchResult.focusOffset, shared_from_this(), offset, direction))```
I feel like `current` might also cause confusion with the current pane i.e. `this`.

Reply:
That's a good idea